contrib/natss/pkg/dispatcher/dispatcher.go

```diff
@@ -182,27 +184,27 @@ func (s *SubscriptionsSupervisor) Connect(stopCh <-chan struct{}) {
 	}
 }
 
-func (s *SubscriptionsSupervisor) UpdateSubscriptions(channel *eventingv1alpha1.Channel, isFinalizer bool) error {
+func (s *SubscriptionsSupervisor) UpdateSubscriptions(channel *eventingv1alpha1.Channel, isFinalizer bool) (*v1alpha1.SubscribableStatus, error) {
```

Comment:
I know it didn't have any, but can you add a comment to this method as it will be useful for future reference?